examples/quickstart/print_to_console.cpp

```diff
@@ -21,20 +22,16 @@ void print(T const& t)
 // define a plain action encapsulating the print function above
 template <typename T>
 struct print_action
-  : hpx::actions::make_action<void (*)(T const&), &print<T>, print_action<T> >
+  : hpx::actions::make_action<void (*)(T const&), &print<T>, print_action<T> >::type
 {};
 
-HPX_REGISTER_PLAIN_ACTION_TEMPLATE((template <typename T>), (print_action<T>));
-
 // define a direct action which is semantically equivalent to the plain action
 // above
 template <typename T>
 struct print_direct_action
-  : hpx::actions::make_direct_action<void (*)(T const&), &print<T>, print_direct_action<T> >
+  : hpx::actions::make_direct_action<void (*)(T const&), &print<T>, print_direct_action<T> >::type
 {};
```

Comment:
It's not really necessary. I needed to do that change during development and forgot to revert. Action access with nested `::type` is not a problem.